bin/hyperlog.js

```diff
@@ -173,7 +179,7 @@ const main = async () => {
     const client = new HyperCoreLogReader(storage, key, null, null, streamOpts)
 
     client.on('data', async (data) => {
-      const line = data.toString().trimRight()
+      const line = data.toString().trimRight().replace(prefixRegExp, '')
```

Comment:
hmm what if prefix is null? maybe we could switch it to something like this:
```js
let line = data.toString().trimRight() 
if (prefixRegExp) line = line.replace(prefixRegExp, '')
```

Reply:
@vigan-abd hah, exactly. `replace` casts the first parameter to a string